tests/store/artifact/test_sftp_artifact_repo.py

```diff
@@ -170,3 +170,74 @@ def test_log_artifacts():
 
             with open(posixpath.join(remote_dir, directory, file2), "rb") as remote_content:
                 assert remote_content.read() == file_content_2
+
+
+@pytest.mark.requires_ssh
+def test_delete_artifact():
+    for artifact_path in [None, "sub_dir", "very/nested/sub/dir"]:
```

Comment:
Instead of a for-loop, how about parametrizing the test? https://docs.pytest.org/en/6.2.x/example/parametrize.html

ditto below

Reply:
done